### ISSUE TITLE
Better gas estimations

### ISSUE THREAD

**Issue by Atmosfearful** [KlimaDAO/klimadao#519]
The metamask gas calculation is not always correct, which can cause transactions to fail

Solution:
- when the app loads, use https://polygonscan.com/gastracker to query the gasPrice for "Rapid" transactions
- add 10%
- store this value as state property in redux
- for all transaction method calls, pass this value down as an override
`contract.retireCarbon(...args, { gasPrice })`
 https://docs.ethers.io/v5/api/contract/contract/#Contract-functionsCall
 
# Other ideas
Ping the api before each transaction and keep the value in-scope rather than saving it in redux?

**Comment by biwano** [comment]
I am reopening this to follow up on this comment: https://github.com/KlimaDAO/klimadao/pull/541#pullrequestreview-1073205754

**Comment by biwano** [comment]
- [ ] let's only do this for the offset retirement transaction. I've realized the others like Stake and wrap etc. fail much more rarely. Another reason I only want to try on the offset aggregator for now is I noticed values fluctuate wildly from the gas station so I am skeptical that this will really help.
- [ ] For the offset view, fallback to `60 gwei` if the gas station API fails.
- [ ] Keep the 10% increase as before
- [ ] NOTE that the property names are different `maxFee (gas station) -> maxFeePerGas (ethers)`
- [ ] I recommend just close this PR and start fresh from the latest staging (sorry!)
- [ ] be sure to update the other ones that we already changed on production
